integrationTests/vm/iele/vmDeploy_test.go

```diff
@@ -37,8 +37,7 @@ func TestVMInvalidSmartContractCodeShouldNotGenerateAccount(t *testing.T) {
 	}
 
 	// tx is not processed due to the invalid sc code
-	err := txProc.ProcessTransaction(tx, round)
-	assert.NotNil(t, err)
+	_ = txProc.ProcessTransaction(tx, round)
```

Comment:
Why the remove for the error test?

Reply:
because it does not return error anymore.